app/diagnostics.go

```diff
@@ -538,6 +539,10 @@ func (a *App) trackConfig() {
 		"isdefault_custom_url_schemes": len(*cfg.DisplaySettings.CustomUrlSchemes) != 0,
 	})
 
+	a.SendDiagnostic(TRACK_CONFIG_SIDEBAR, map[string]interface{}{
```

Comment:
Actually, can you move this under the TRACK_CONFIG_SERVICE section? I think their location in the diagnostics matches their location in the config

Reply:
That's fine.  I'll make the change